server/utils_test.go

```diff
@@ -78,3 +78,51 @@ func TestParseOwnerAndRepo(t *testing.T) {
 		assert.Equal(t, repo, tc.ExpectedRepo)
 	}
 }
+
+func TestIsFlag(t *testing.T) {
+	tcs := []struct {
+		Text     string
+		Expected bool
+	}{
+		{Text: "--test-flag", Expected: true},
+		{Text: "--testFlag", Expected: true},
+		{Text: "test-no-flag", Expected: false},
+		{Text: "testNoFlag", Expected: false},
+		{Text: "test no flag", Expected: false},
+	}
+
+	for _, tc := range tcs {
+		assert.Equal(t, tc.Expected, isFlag(tc.Text))
+	}
+}
+
+func TestParseFlag(t *testing.T) {
+	tcs := []struct {
+		Text     string
+		Expected string
+	}{
+		{Text: "--test-flag", Expected: "test-flag"},
+		{Text: "--testFlag", Expected: "testFlag"},
+		{Text: "testNoFlag", Expected: "testNoFlag"},
+	}
+
+	for _, tc := range tcs {
+		assert.Equal(t, tc.Expected, parseFlag(tc.Text))
+	}
+}
+
+func TestContainsValue(t *testing.T) {
+	tcs := []struct {
+		List     []string
+		Value    string
+		Expected bool
+	}{
+		{List: []string{"value1", "value2"}, Value: "value1", Expected: true},
+		{List: []string{}, Value: "value1", Expected: false},
+		{List: []string{"value1", "value2"}, Value: "value1", Expected: true},
```

Comment:
Should this case be removed since it looks to be the same as on line 120? unless you wanted to test `value2` as opposed to `value1`?

Reply:
Updated!